lib/releases.js

```diff
@@ -80,74 +79,84 @@ const contributorsSentence = ({ commits, pullRequests }) => {
 }
 
 const getCategoriesConfig = ({ config }) => {
-  const categoriesConfig = config.categories.reduce((acc, category) => {
-    const labels = category.label ? [category.label] : category.labels
-    labels.forEach(label => (acc[label] = category))
-    return acc
-  }, {})
-
-  const orderedCategories = [
-    UNCATEGORIZED,
-    ..._.flatMap(config.categories, category =>
-      category.label ? category.label : category.labels
-    )
-  ]
-
-  return [categoriesConfig, orderedCategories]
+  const categoriesConfig = [{ title: UNCATEGORIZED }, ...config.categories]
+
+  return categoriesConfig.map(category => {
+    if (!category.labels) category.labels = []
+    if (category.label) {
+      // treat label as labels
+      category.labels.push(category.label)
+      delete category.label
+    }
+    return { ...category, pullRequests: [] }
+  })
 }
 
-const categorizePullRequests = (pullRequests, categories, excludeLabels) => {
-  return pullRequests.reduce(
-    (acc, pullRequest) => {
+const categorizePullRequests = (
+  pullRequests,
+  orderedCategories,
+  excludeLabels
+) => {
+  const categories = orderedCategories.flatMap(category => category.labels)
```

Comment:
Categories and labels are not the same thing. I'm not sure if my suggestion is the best name, but something along these lines would be an improvement.

```suggestion
  const allCategoryLabels = orderedCategories.flatMap(category => category.labels)
```